core/lang/cs.ini

```diff
@@ -8,3 +8,6 @@ no named subcommand = Příkaz **((name))** nemá podpříkaz **((subcommand))**
 
 [create_embed]
 footer = Odpověď pro
+
+[version]
+reply = Příkaz není dostupný
```

Comment:
I think we should say something a bit different:
```diff
- reply = Příkaz není dostupný
+ reply = Příkaz není v této verzi dostupný
+ reply = Příkaz není v této verzi discord.py dostupný
+ reply = To teď neumím
```
or something else, feel free to improvise.